docs/DeveloperGuide.md

```diff
@@ -377,55 +367,62 @@ public void addManager(Scanner scanner, WeekView weekView, MonthView monthView,
 - `taskTypeString`: String representing the task type.
 - `taskDescription`: Description of the task.
 
-### Handling User Input
+#### Exceptions
+
+- `TaskManagerException`: Thrown if there is an error in managing tasks.
+- `DateTimeParseException`: Thrown if there is an error parsing the date or time.
+
+#### Handling User Input
```

Comment:
Great job on adding these exceptions to make it consistent with the rest of the DG!